Resize (Resolution Node): image is not resized if width/height factor > 256

**Information:**
- Chainner version: 0.19.2
- OS: Windows 11

**Description**
- When width/height ratio or height/width **ratio is greater than 256**, the image is not generated. Except for width or height equal to **1, ratio is 308**
- It seems that this issue is related to PIL: not sure if this is a bug or a limitation.
![resize-resolution](https://github.com/chaiNNer-org/chaiNNer/assets/137266603/ce29ab15-901e-4cc8-aad5-ec28fdb0f106)

**Logs**
`[2023-08-15 14:23:27 +0200] [360] [ERROR] Error broadcasting output: height and width must be > 0`

--- a/backend/src/nodes/properties/outputs/numpy_outputs.py
+++ b/backend/src/nodes/properties/outputs/numpy_outputs.py
@@ -10,7 +10,7 @@ from nodes.base_output import BaseOutput, OutputKind
 from ...impl.image_utils import normalize, to_uint8
 from ...impl.pil_utils import InterpolationMethod, resize
 from ...utils.format import format_image_with_channels
-from ...utils.utils import get_h_w_c
+from ...utils.utils import get_h_w_c, round_half_up
 
 
 class NumPyOutput(BaseOutput):
@@ -129,7 +129,7 @@ def preview_encode(
     max_size = target_size * grace
     if w > max_size or h > max_size:
         f = max(w / target_size, h / target_size)
-        t = (int(w / f), int(h / f))
+        t = (max(1, round_half_up(w / f)), max(1, round_half_up(h / f)))
         if c == 4:
             # https://github.com/chaiNNer-org/chaiNNer/issues/1321
             img = resize(img, t, InterpolationMethod.BOX)

